packages/engine-server/src/DendronEngineV3.ts

```diff
@@ -172,12 +172,15 @@ export class DendronEngineV3 extends EngineV3Base implements DEngine {
    * Does not throw error but returns it
    */
   async init(): Promise<DEngineInitResp> {
+    const { data: config } = DConfig.readConfigAndApplyLocalOverrideSync(
+      this.wsRoot
+    );
```

Comment:
I'm not sure if it is safe to use `readConfigAndApplyLocalOverrideSync` yet.
We've recently identified where this causes a problem (where overrides were being merged back into `dendron.yml`) and fixed it, but there still may be places where that still happens. (related wip task: [[Make Sure Config Reads and Writes Dendronyml Correctly|dendron://private/task.2022.10.02.make-sure-config-reads-and-writes-dendronyml-correctly]])

It might be safe to keep this as-is or not apply overrides.

Reply:
(same with all config read changes in this branch)

Reply:
Oh got it, thanks for letting me know. I forget if one of the tests depends on the local overrides, but that can be worked around